test/parameter.js

```diff
@@ -177,4 +120,55 @@ describe('Parameter to Member converter', () => {
     expect(enumerations).to.be.instanceof(minim.elements.Array);
     expect(enumerations.toValue()).to.deep.equal([['hello']]);
   });
+
+  it('creates a warning when example does not match parameter type', () => {
+    const parser = new Parser({ minim, source: '' });
+    parser.result = new minim.elements.ParseResult();
+    parser.convertParameterToMember({
+      type: 'string',
+      'x-example': 5,
```

Comment:
So this would result in dropping the parameter? If people were relying in numbers being coerced to strings, this could result in quite a breaking change.